errata_tool/connector.py

```diff
@@ -6,18 +6,38 @@
 import re
 import time
 import six
+import krbV
 
 
 class ErrataConnector(object):
     # Staging is https://errata.stage.engineering.redhat.com
     _url = "https://errata.devel.redhat.com"
     _auth = HTTPKerberosAuth(mutual_authentication=DISABLED)
+    _username = None
     ssl_verify = True  # Shared
     debug = False
 
     # Timings are only recorded if debug is set to True
     timings = {'GET': {}, 'POST': {}, 'PUT': {}}
 
+    def _set_username(self, **kwargs):
+        if self._username is not None:
+            return
+        try:
+            ctx = krbV.default_context()
+            cache = ctx.default_ccache()
+            # XXX What if you have >1 ticket?
+            self._username = cache.principal()[0]
+        except krbV.Krb5Error, IndexError:
+            raise ErrataException('Pigeon crap. Did it forget to run kinit?')
+
+    def getUser(self, **kwargs):
+        self._set_username()
+        name = self._username
+        if 'username' in kwargs:
+            name = kwargs['username']
+        return self._get(self._url + '/api/v1/user/' + name)
```

Comment:
There's a new `User` class that loads this same `/api/v1/user` URL this now, could we use that?

Reply:
yup.